ports/stm32/README.md

```diff
@@ -53,7 +53,10 @@ The default board is PYBV10 but any of the names of the subdirectories in the
 should produce binary images in the `build-PYBV11/` subdirectory (or the
 equivalent directory for the board specified).
 
-You must then get your board/microcontroller into DFU mode.  On the pyboard
+### Flashing the Firmware using DFU mode
+
+You must then get your board/microcontroller into DFU (Device Firmware
+Update) mode.  On the pyboard
```

Comment:
To make this more general for any stm32 board, how about putting the part about `machine.bootloader()` here, something like: "If you already have MicroPython installed on the board then call `machine.bootloader()` on the board, either at the REPL or using pyboard.py...".

Then "Alternatively, or if you don't have MicroPython installed, then enter DFU mode via hardware.   On the PYBv1.x connect the 3V3 pin ..."

Then could put the part about pyboard-D here as well, before moving on to the existing text describe how to do "make deploy".

Reply:
done, needs to be squashed in merge